microsoft-edge/progressive-web-apps-chromium/how-to/digital-goods-api.md

```diff
@@ -0,0 +1,213 @@
+## Digital Goods API
+
+The Digital Goods API is an interface between your PWA app and Microsoft Store.  The Digital Goods API supports:
+* Querying the details of a digital item from the Microsoft Store backend, such as the item's name, description, and regional price.
+* Consuming or acknowledging purchases.
+* Checking the digital items that are currently owned by the user.
+* Checking the purchase history of the user.
+
+See:
+* [Digital Goods API For Microsoft Store PWA Explainer](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/PwaDigitalGoods/explainer.md)
+* [Digital Goods API: Draft Community Group Report](https://wicg.github.io/digital-goods/)
+
+
+<!-- ====================================================================== -->
+## Payment Request API
+
+The Payment Request API⁠⁠ handles the actual payment transaction when a purchase is made by user.  The Payment Request API uses the item details that the Digital Goods API provides, to make the in-app purchase by using whichever billing payment method the user has set up at the Microsoft Store.
+
+See [Payment Request API](https://www.w3.org/TR/payment-request/) at W3C.
+
+
+<!-- ====================================================================== -->
+## Checking whether the Digital Goods API is available
+
+To detect whether you've correctly enabled the API on your website, check for the `getDigitalGoodsService` method in the window object:
+
+```javascript
+if ('getDigitalGoodsService' in window) {
+  // Digital Goods API is supported!
+} else {
+  console.log('DigitalGoodsService is not available.');
+  // Use other payment method
+}
+```
+
+
+<!-- ====================================================================== -->
+## Connecting to the Microsoft Store Billing service (`getDigitalGoodsService` method)
+
+Use the `getDigitalGoodsService` method to connect to the Microsoft Store Billing service.
+
+The Digital Goods API was designed to be compatible with various browsers and digital stores, similar to how the Payment Request API is browser-agnostic and can be used with different payment providers.  To retrieve an instance of the service for Microsoft Store Billing, pass the string `"https://store.microsoft.com/billing"` as the payment method to the `getDigitalGoodsService` method.
+
+If the method throws an error, the Microsoft Store Billing payment method is not available (such as when the user is accessing your PWA through the browser).  Alternatively, consider providing a different payment method for transactions.
+
+```javascript
+if (window.getDigitalGoodsService === undefined) {
+  // Digital Goods API is not supported in this context.
+  return;
+}
+try {
+  const digitalGoodsService = await window.getDigitalGoodsService("https://store.microsoft.com/billing");
+  // Use the service here.
+  ...
+} catch (error) {
+  // Our preferred service provider is not available.
+  // Use a normal web-based payment flow.
```

Comment:
```suggestion
  // Use a web-based payment flow instead.
```

I don't think "normal" is warranted here, it implies the digitial goods API flow is not normal.